rust/azure_iot_operations_services/src/azure_device_registry/client.rs

```diff
@@ -514,6 +516,7 @@ where
             .get_device_command_invoker
             .invoke(get_device_request)
             .await
+            .map_err(ErrorKind::from)?
```

Comment:
what changed here to need an extra error mapping?

Reply:
This is for the service error i think

Reply:
Codegen Invoker now returns Result<Result<Response, AkriServiceError>, AIOProtocolError>.

Basically, the placeholder `ServiceError` is no longer required - codegen gives us a real one now.

Reply:
> u might need to update docs if there is any other error

Done